www/choosing-install-method.rst

```diff
@@ -0,0 +1,149 @@
+  web frameworks and other popular Python packages.
+- Also includes important non-Python packages (like HDF5, Qt, gcc, R, etc.) that
+  are often hard to install by other means.
+- Binaries for everything, usually no compiler needed.
+
+Cons:
+
+- There may not be an installer if you're on a less commonly used OS
+  (32-bit Linux for example).
+- Doesn't integrate with an existing Python installation.
+
+Using an OS-specific package manager
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Pros:
+
+- Package manager itself already installed (on Linux) or so useful for
+  non-Python things that you'll want to install it anyway (on OS X).
+- Package managers are reliable, mature technology.
+- Easy to get started.
+
+Cons:
+
+- On Linux, installs into the same Python environment as used by the OS itself,
+  which can be fragile if you want to install packages also with pip_.
+- On OS X, packages provided by Homebrew and Macports are often not as well
+  tested or optimized as other install methods.
+- Not everything you need will be packaged.
+
+Using ``pip``
+~~~~~~~~~~~~~
+Pros:
+
+- Can work with an existing Python install (do *not* install into the Python
+  provided by your OS though, that's always a bad idea).
+- Is very widely used, and is the installer recommended by the
+  `Python Packaging Authority <https://www.pypa.io>`_.
+- Installs wheels, which more and more packages provide on PyPI_.
+
+Cons:
+
+- Does not work well on Windows.
+- Does not include a true dependency resolver (so state can become inconsistent).
+- Cannot handle non-Python dependencies (those are important for scientific
+  applications: BLAS/LAPACK, HDF5, etc.).
+- Not all packages provide complete dependency info on PyPI, which can make
+  pip_ installs fail.
+- Publishing wheels on PyPI (so pip_ can install them) is hard, especially
+  for packages with compiled code.
+
+Using ``conda``
+~~~~~~~~~~~~~~~
+Note: conda_ is the *package manager* on which the Anaconda_ and Miniconda_
+*distributions* are built.  For a good in-depth discussion on what conda_
+is and is not, see `this blog post
+<https://jakevdp.github.io/blog/2016/08/25/conda-myths-and-misconceptions/>`_.
+
+Pros:
+
+- Can handle non-Python dependencies.
+- Has a true dependency resolver.
+- Works well on Windows, OS X and Linux.
+- Can manage multiple Python versions without issues.
+- Conda environments have better isolation that virtualenvs.
+- Using pip_ within conda_ environments works well.
+- Publishing conda_ packages is a streamlined process (with `conda-forge`_).
+
+Cons:
+
+- Cannot work with an existing Python install.
+- Cannot use wheels (use conda binary packages instead), so for projects
+  not yet packaged for conda_ one still needs to use pip_ (or do the
+  packaging, which is fortunately easy).
+- Documentation and usage instructions are often sparser than for pip_.
+- If you're on a less commonly used OS (like 32-bit Linux), there will be
+  no precompiled packages.
+
```

Comment:
I'd also add that users have to trust the source of the pre-compiled binaries (by a company). That may not be a concern for most users, but that's a huge security concern for others. Some feel that's a reason big enough not to use conda, which isn't reflected here.